src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  margin-top: 16px;
+  margin-bottom: 24px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:nth-child(-n + 4) {
```

Comment:
The `&:nth-child(-n + 4)` selector is used to apply styles to the first 4 stars, assuming that there will always be 4 active stars. However, this does not provide flexibility if the number of active stars needs to change. Consider adding a modifier class for each star rating (e.g., `.stars--4 &__star:nth-child(-n+4)`), which allows you to control the number of active stars through the class applied to the `.stars` block.